llvm/test/CodeGen/WebAssembly/wide-arithmetic.ll

```diff
@@ -0,0 +1,51 @@
+; RUN: llc -mattr=+wide-arithmetic < %s | FileCheck %s
```

Comment:
It would be useful to use the auto-update script (utils/update_llc_test_checks.py) here. It will add more test output, but I think that will actually be useful, since it's valuable to see not just that the expected instruction is emitted, but to see that the context looks correct as well.

Reply:
Can do 👍